Automattic/jetpack
# RNA: Components package

#### Changes proposed in this Pull Request:
* The RNA Components package initial structure

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Rebuild Jetpack: `jetpack build plugins/jetpack`.
2. Go to Jetpack Dashboard and confirm that you see the Jetpack logo in the top left corner.
3. Rebuild Connection UI: `jetpack build packages/connection-ui`.
4. Go to "Tools -> Connection Manager" and confirm that you see the Jetpack logo in the top left corner.
